semainda/store-manager
# store-manager-development

**What does this PR do?**
Have Store Manager UI templates

**Description of Task completed?**
Have the following templates working

- Administrator templates
Login
Dashboard
Add product category
Edit product category
List all product categories
Add product sub-category
Add product
Edit product
List all stock products
Add role
Assign user role
Edit role
List all roles
Add users
List all store users
Add a sale record
List all sales records

- Attendant templates
Login
Dashboard
Add product
Edit product
List all stock products
Add a sale record
List all own sales records

**How should this be manually tested?**

One should  do the following to test it
- Install the browser
- Create a project directory
- cd to the project directory created above
- Clone the store-manager repository
- cd to the store-manager directory
- Double click the index.html page to get started with the application.

**What is the relevant pivotal tracker stories link?**
https://www.pivotaltracker.com/n/projects/2203903
